cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -86,87 +85,97 @@ Kitematic and then download, start, and connect to an instance of CDAP.
+     :mapping: linux,windows
+     :dependent: linux-windows
+     :languages: console,shell-session
+     
+     $ docker-machine create --driver virtualbox cdap
+     $ docker-machine env cdap
+     
+   This will create a new Docker virtual machine named ``cdap`` and print out the environment.
+
+#. When you run ``docker-machine env cdap``, it will print a message on the screen such as::
+
+     export DOCKER_TLS_VERIFY="1"
+     export DOCKER_HOST="tcp://192.168.99.100:2376"
+     export DOCKER_CERT_PATH="/Users/<username>/.docker/machine/machines/cdap"
+     export DOCKER_MACHINE_NAME="cdap"
+     # Run this command to configure your shell: 
+     # eval $(docker-machine env cdap)
+ 
+   It is essential to run these export commands (or the single ``eval`` command). Otherwise,
+   subsequent Docker commands will fail because they won't be able to connect to the correct
+   Docker VM.
   
-  When you run ``boot2docker start``, it will print a message on the screen such as::
-
-    To connect the Docker client to the Docker daemon, please set:
-        export DOCKER_HOST=tcp://192.168.59.103:2375
-        export DOCKER_CERT_PATH=/Users/.../.boot2docker/certs/boot2docker-vm
-        export DOCKER_TLS_VERIFY=1
-
-  It is essential to run these export commands (or command, if only one). Otherwise,
-  subsequent Docker commands will fail because they can't tell how to connect to the
-  Docker VM.
-
-- Once Docker has started, pull down the *CDAP Docker Image* from the Docker hub using:
-
-  .. tabbed-parsed-literal::
-    :tabs: "Mac OS X",Windows
-    :mapping: linux,windows
-    :dependent: linux-windows
-    :languages: console,shell-session
-
-    .. Mac OS X
-    
-    $ docker pull caskdata/cdap-standalone:|release|
-    
-    .. Windows
-    
-    > docker pull caskdata/cdap-standalone:|release|
-
+#. If you are running **Docker on either Mac OS X or Microsoft Windows**, Docker is running a
+   virtual Linux machine on top of your host OS. You will need to use the address shown
+   above (such as ``192.168.99.100``) as the host name when either connecting to the CDAP
+   UI or making an HTTP request.
+
+#. Once Docker has started, pull down the *CDAP Docker Image* from the Docker Hub using:
+
+   .. tabbed-parsed-literal::
+     :tabs: "Linux or Mac OS X",Windows
+     :mapping: linux,windows
+     :dependent: linux-windows
+     :languages: console,shell-session
+ 
+     $ docker pull caskdata/cdap-standalone:|release|
 
-- Start the *Docker CDAP Virtual Machine* with:
+#. Start the *Docker CDAP Virtual Machine* with:
 
-  .. tabbed-parsed-literal::
-    :tabs: "Mac OS X",Windows
-    :mapping: linux,windows
-    :dependent: linux-windows
-    :languages: console,shell-session
+   .. tabbed-parsed-literal::
+     :tabs: "Linux or Mac OS X",Windows
+     :mapping: linux,windows
+     :dependent: linux-windows
+     :languages: console,shell-session
+ 
+     $ docker run -t -i -p 9999:9999 -p 10000:10000 caskdata/cdap-standalone:|release|
```

Comment:
Let's change this:

`docker run -d --name cdap-standalone -p 9999:9999 -p 10000:10000 caskdata/cdap-standalone:|release|`

This will name the container `cdap-standalone` which we can reference below, versus having the Docker daemon auto-generating a name.